crates/bitwarden-crypto/src/enc_string/symmetric.rs

```diff
@@ -30,17 +31,20 @@ export type EncString = string;
 /// variants, but we should be opinionated in which variants are used for encrypting.
 ///
 /// ## Variants
-/// - [Aes256Cbc_B64](EncString::Aes256Cbc_B64)
+/// - [Aes256Cbc_B64](EncString::Aes256Cbc_B64) - Deprecated and MUST NOT be used for encrypting as
+///   it is not authenticated
 /// - [Aes256Cbc_HmacSha256_B64](EncString::Aes256Cbc_HmacSha256_B64)
+/// - [Cose_Encrypt0_B64](EncString::Cose_Encrypt0_B64) - The preferred variant for encrypting data.
 ///
 /// ## Serialization
 ///
-/// [EncString] implements [Display] and [FromStr] to allow for easy serialization and uses a
+/// [EncString] implements [ToString] and [FromStr] to allow for easy serialization and uses a
 /// custom scheme to represent the different variants.
 ///
 /// The scheme is one of the following schemes:
 /// - `[type].[iv]|[data]`
 /// - `[type].[iv]|[data]|[mac]`
+/// - `[type].[data]`
```

Comment:
This would be misleading? Since COSE is not encrypted data but rather a binary representation of a complex data mode.

Reply:
Updated to `[type].[cose_encrypt0_bytes]`, with a note saying it's the Cose Encrypt0 message, serialized.